src/server/managers/AppListenerManager.ts

```diff
@@ -578,4 +583,24 @@ export class AppListenerManager {
             }
         }
     }
+
+    // Livechat
+    private async executeLivechatRoomClosed(data: ILivechatRoom): Promise<void> {
+        const cfLivechatRoom = Utilities.deepCloneAndFreeze(data);
+
+        for (const appId of this.listeners.get(AppInterface.ILivechatRoomClosedHandler)) {
+            const app = this.manager.getOneById(appId);
+
+            if (!app.hasMethod(AppMethod.EXECUTE_LIVECHAT_ROOM_CLOSED_HANDLER)) {
+                return;
+            }
+
+            return app.call(AppMethod.EXECUTE_LIVECHAT_ROOM_CLOSED_HANDLER,
```

Comment:
This `return` statement is going to prevent multiple apps to handle this events - is this the expected behavior?

Reply:
Sorry, it's my fault. It should be `continue` here 😅

Reply:
early return 🙅
early continue 👈